rustls/src/crypto/mod.rs

```diff
@@ -110,7 +110,7 @@ pub use crate::suites::CipherSuiteCommon;
 ///
 /// # Making a custom `CryptoProvider`
 ///
-/// Your goal will be to populate a [`crypto::CryptoProvider`] struct instance.
+/// Your goal will be to populate an instance of this [`crypto::CryptoProvider`] struct.
```

Comment:
I would like to do this update a little differently, thankfully it missed the merge queue. I would like to remove the brackets, as it seems pretty useless for this page to reference itself here. Using `CryptoProvider` with no brackets also seems more consistent with how this is done elsewhere for `CryptoProvider` in the documentation. PR should be coming soon.